controller/test/sensor_tests/SensorTests.cpp

```diff
@@ -154,3 +156,12 @@ TEST(SensorTests, FullScaleReading) {
         << "Patient Sensor at index" << i;
   }
 }
+
+//@TODO: Write complete unit tests for the volumetric flow calculation
```

Comment:
I think it would be good to get if not "all of them" done, a few more.  For instance a test with a negative pressure, a test with 0 pressure, a test with a very large pressure and a very small pressure.

Reply:
Will do - wanted to get something out for eyes to see.